lib/git-interface/pull_request.d.ts

```diff
@@ -0,0 +1,55 @@
+import { Issue } from "./issue"
+
+export interface PullRequest extends Issue
```

Comment:
future-todo: gitlab has `Merge Requests` instead of PRs

Reply:
I don't know if this is super necessary. At the end of the day, `Issue` and `PullRequest` are interfaces. We'd have separate implementations for GitHub and GitLab. The naming difference could be resolved there.